src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import { getUserById } from '../../services';
```

Comment:
Avoid using the `import` keyword in non-root components like `TodoInfo`. Instead, prepare data in the App component and pass it down as props. This aligns with the checklist guideline: 'Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components.'